### PR TITLE
Integration Tests for Service methods

### PR DESCRIPTION
**Pull request by LiashenkoAndrey**
ticket - https://lizavol9.atlassian.net/browse/KAN-66?atlOrigin=eyJpIjoiYjQ2ZjIyMzZlMTY2NDZiNmI3ZWRlNDEyNzU3MDRhMDkiLCJwIjoiaiJ9

### CODE REVIEW

File: src/test/java/online/talkandtravel/service/impl/integrationtest/MessageServiceIntegrationTest.java [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+    SendMessageRequest request = new SendMessageRequest("content", arubaChatId, null);
+
+    assertThrows(UserNotJoinedTheChatException.class, () -> underTest.saveMessage(request, principal));
+  }
+
+  @Test
+  void saveMessage_shouldThrow_whenChatNotFound() {
+    Principal principal = testAuthenticationService.authenticateUser(bob);
+    Long notExistingChatId = 777L;
```
Comment (OleksandrProkopenkoDev) [inline src/test/java/online/talkandtravel/service/impl/integrationtest/MessageServiceIntegrationTest.java:63]: here we can create 'ChatTestData' class, and place there as private all these constants, and make public static methods to getConstant ( getNotExistingChatId() )

File: src/test/java/online/talkandtravel/service/impl/integrationtest/MessageServiceIntegrationTest.java [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    SendMessageRequest request = new SendMessageRequest("content", notExistingChatId, null);
+
+    assertThrows(UserNotJoinedTheChatException.class, () -> underTest.saveMessage(request, principal));
+  }
+
+  @Test
+  void saveMessage_shouldThrow_whenRepliedMessageNotFound() {
+    Principal principal = testAuthenticationService.authenticateUser(bob);
+    Long repliedMessageId = 1000L;
```
Comment (OleksandrProkopenkoDev) [inline src/test/java/online/talkandtravel/service/impl/integrationtest/MessageServiceIntegrationTest.java:72]: same

File: src/test/java/online/talkandtravel/service/impl/integrationtest/MessageServiceIntegrationTest.java [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+    SendMessageRequest request = new SendMessageRequest("content", arubaChatId, repliedMessageId);
+
+    assertThrows(WebSocketException.class, () -> underTest.saveMessage(request, principal));
+  }
+
+  @Test
+  void saveMessage_shouldThrow_whenRepliedMessageFromAnotherChat() {
+    Principal principal = testAuthenticationService.authenticateUser(bob);
+    Long angolaChatId = 3L;
```
Comment (OleksandrProkopenkoDev) [inline src/test/java/online/talkandtravel/service/impl/integrationtest/MessageServiceIntegrationTest.java:81]: same

File: src/test/java/online/talkandtravel/service/impl/integrationtest/MessageServiceIntegrationTest.java [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+
+  @Autowired
+  private TestAuthenticationService testAuthenticationService;
+
+  @Autowired
+  private MessageService underTest;
+
+  private User bob, alise;
+
+  private static final Long arubaChatId = 1L, existingMessageIdOfArubaChat = 1001L;
```
Comment (OleksandrProkopenkoDev) [inline src/test/java/online/talkandtravel/service/impl/integrationtest/MessageServiceIntegrationTest.java:43]: move this into 'ChatTestData' class

File: src/test/java/online/talkandtravel/service/impl/integrationtest/MessageServiceIntegrationTest.java (hunk 1 above)
Comment (LiashenkoAndrey) [reply src/test/java/online/talkandtravel/service/impl/integrationtest/MessageServiceIntegrationTest.java:63]: I've created public static constants in the ChatTestData class. In my opinion, there's no need to create additional getters for these constants.

**Review by OleksandrProkopenkoDev — APPROVED** [review]

**Pull request merged**